src/dev/economy_passive.rb

```diff
@@ -253,6 +253,6 @@ def image?(message)
   # @return [bool] Has link?
   def link?(message)
     return false if message.nil? or message.content.nil?
-    return message.content =~ /#{URI::regexp(['http', 'https'])}/
+    return message.content =~ /#{URI::DEFAULT_PARSER.make_regexp(['http', 'https'])}/
```

Comment:
please revert 256 to `return message.content =~ /#{URI::regexp(['http', 'https'])}/`

Reply:
ok